examples/gno.land/p/demo/gnorkle/feeds/static/feed.gno

```diff
@@ -0,0 +1,143 @@
+package static
+
+import (
+	"bufio"
+	"bytes"
+
+	"gno.land/p/demo/gnorkle/feed"
+	"gno.land/p/demo/gnorkle/gnorkle"
+	"gno.land/p/demo/gnorkle/ingesters/single"
+	"gno.land/p/demo/gnorkle/message"
+	"gno.land/p/demo/gnorkle/storage/simple"
+	"gno.land/p/demo/ufmt"
+)
+
+// Feed is a static feed.
+type Feed struct {
+	id            string
+	isLocked      bool
+	valueDataType string
+	ingester      gnorkle.Ingester
+	storage       gnorkle.Storage
+	tasks         []feed.Task
+}
+
+// NewFeed creates a new static feed.
+func NewFeed(
+	id string,
+	valueDataType string,
+	ingester gnorkle.Ingester,
+	storage gnorkle.Storage,
+	tasks []feed.Task,
+) *Feed {
+	return &Feed{
+		id:            id,
+		valueDataType: valueDataType,
+		ingester:      ingester,
+		storage:       storage,
+		tasks:         tasks,
+	}
+}
+
+// NewSingleValueFeed is a convenience function  for creating a static feed
+// that autocommits a value after a single ingestion.
+func NewSingleValueFeed(
+	id string,
+	valueDataType string,
+	tasks []feed.Task,
```

Comment:
maybe variadic (`tasks ...feed.Task`) is nicer

Reply:
Yeah I think you're right.  When writing the realm code that initializes the feed, this can cut out the middle step of having to put each task in a slice. I'll make this change.